packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -589,6 +589,10 @@ const AccountSwitcher = props => {
         canOpenMulti() &&
         (!props.show_eu_related_content || (props.show_eu_related_content && props.can_change_fiat_currency));
 
+    const can_manage_account_when_virtual =
+        props.is_virtual &&
+        (!props.show_eu_related_content || (props.show_eu_related_content && props.can_change_fiat_currency));
```

Comment:
(!props.show_eu_related_content || (props.show_eu_related_content && props.can_change_fiat_currency)); 
This condition is already checked in `can_manage_account.` Do we still need  this condition?

Reply:
@amina-deriv but we have check like `(can_manage_account || can_manage_account_when_virtual)` so I think when `can_manage` is false we still needs to check this in `can_manage_account_when_virtual`

Reply:
you can extract `!props.show_eu_related_content || (props.show_eu_related_content && props.can_change_fiat_currency)` from `can_manage_account`  variable and define a new variable and reuse it in both 
 ```suggestion
        const can_manage_account = (!props.show_eu_related_content || (props.show_eu_related_content && props.can_change_fiat_currency));
        const can_manage_account_virtual = props.is_virtual && can_manage_account;
        const can_manage_account_multi =  !canUpgrade() &&
        canOpenMulti() &&can_manage_account
```